src/test/java/seedu/address/logic/commands/CommandTestUtil.java

```diff
@@ -226,8 +228,10 @@ public static void assertDisplayableCommandSuccess(Command command, Model actual
      * that takes a string {@code expectedMessage} and a Company {@code expectedCompany}.
      */
     public static void assertDisplayableCommandSuccess(Command command, Model actualModel, String expectedMessage,
-                                                       Model expectedModel, Company expectedCompany) {
-        DisplayableCommandResult expectedCommandResult = new DisplayableCommandResult(expectedMessage, expectedCompany);
+                                                       Model expectedModel, Company expectedCompany,
+                                                       String expectedDisplayMessage) {
```

Comment:
Unfortunately due to the feature freeze, I don't think we're allowed to change the UI output :(